components/table/demo/table.html

```diff
@@ -16,28 +16,28 @@
 			<h2>Default</h2>
 			<d2l-demo-snippet>
 				<template>
-					<d2l-test-table type="default"></d2l-test-table>
+					<d2l-test-table type="default" sticky-controls="true"></d2l-test-table>
```

Comment:
These shouldn't need a value since they're boolean attributes?

```suggestion
					<d2l-test-table type="default" sticky-controls></d2l-test-table>
```

Reply:
Good catch, will fix!